src/main/resources/db/changelog/changes/v1.1.0/sql/2021-11-17-01__create_inn_reach_recall_user_table.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE inn_reach_recall_user
+(
+    id                  UUID         NOT NULL,
+    user_id             UUID         NOT NULL,
+    created_by_userid   UUID         NOT NULL DEFAULT '00000000-0000-0000-0000-000000000000',
+    created_by_username VARCHAR(255) NOT NULL DEFAULT 'SYSTEM',
+    updated_by_userid   UUID,
+    updated_by_username VARCHAR(255),
+    created_date        TIMESTAMP    NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    updated_date        TIMESTAMP,
+    CONSTRAINT pk_inn_reach_recall_user PRIMARY KEY (id)
+);
```

Comment:
I believe the model should reference central server by id with `on delete cascade`, as we do for all settings that are per central server. Otherwise when we remove central server config there will be orphaned data

Reply:
if the relation is many-2-one, cascade deletion won't work. so it should be either 1-2-1 or w/o cascade delete